### PR TITLE
Instructions for Circle CI setup

### PR DESCRIPTION
**Pull request by vkurup**
To accompany the Circle CI PR for django-project-template.

### CODE REVIEW

File: source/services/circle-ci.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
+Circle CI
+=========
+
+The project template now has a Circle CI template that you can use for your project. Circle CI's
+benefit over Travis CI is that it works for private non-Caktus repos. Once you have
+completed the initial project setup (i.e. the items in README.rst), do the following:
+
+1. If you don't have one already, `create a Circle CI account <https://circleci.com/>`_, being sure
+   to link it to your Caktus github account.
```
Comment (dpoirier) [inline source/services/circle-ci.rst]: How does one `make sure to link it...`?  I know with some online services it's not obvious how to do that in a way that'll end up working for us.

File: source/services/circle-ci.rst [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+#. Set up HipChat notifications.
+
+   A. Get the Hipchat Room Number. This is the number in the URL bar when you are looking at the
+      room in Hipchat.
+
+   #. Obtain the HipChat API token for Circle CI. Click on the '...' button in the upper right-hand
+      corner of your HipChat room page, then click on 'Integrations...', then click on the
+      'Installed' button to filter by already-installed integrations. You should see Circle CI
+      listed there. Click on it and you will find the Token.
```
Comment (dpoirier) [inline source/services/circle-ci.rst]: Just wanted to ask if you have HipChat admin? If you do, we might want someone without admin to test that they can get the token this way.  (I can't test as I do have admin.)

Comment (vkurup) [inline source/services/circle-ci.rst]: I *think* I'm an admin, so I added the caveat about possibly needing to create a Caktus tech support request (copied from the travis.rst instructions)

**Comment by vkurup** [comment]
Good feedback. This is updated now.

**Comment by dpoirier** [comment]
:shipit:

**Pull request merged**